src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -560,6 +563,33 @@ static void read_channel_args(grpc_chttp2_transport* t,
           .value_or(true);
 }
 
+void grpc_chttp2_transport::WriteSecurityFrame(grpc_core::SliceBuffer* data) {
+  combiner->Run(grpc_core::NewClosure([this, data](grpc_error_handle) mutable {
```

Comment:
I think we need to Ref() the transport here so that we don't delete the transport while waiting for the lock to run

Reply:
Done, thanks.